### ISSUE TITLE
Feature request: show all items before repeating

### ISSUE THREAD

**Issue by cristoper** [LouisDotCom/NextItem#3]
It might be nice if the app showed all items at least once (in a random order) before repeating any item.

Or, display items based on a weighted frequency and allow the user to set a priority for each item.

**Comment by LouisDotCom** [comment]
Yes. Otherwise, getting three 5k runs in a row might make the one think he is being reduced to a pretty dull schedule. Perhaps make each item equal weight by default and, when an item appears, allow the user to click "ok" or "increase priority" or "decrease priority"
